boards/posix/native_posix/native_posix_defconfig

```diff
@@ -5,5 +5,4 @@ CONFIG_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=1000000
 CONFIG_COVERAGE=y
-CONFIG_FAKE_ENTROPY_NATIVE_POSIX=y
```

Comment:
What about setting `CONFIG_ENTROPY_GENERATOR=y` before setting `CONFIG_FAKE_ENTROPY_NATIVE_POSIX=y` here instead?

Dependencies in Kconfig must be explicitly enabled (because the generality makes it impossible to guess how people want to resolve dependencies in many cases).

Reply:
Well.. the thing is that we do not want to build always the entropy driver or the BT stack.
But if the project selects it, we want this options to be built by default.
So the behaviour of the old code was what we wanted.
This is just to avoid the warnings.

Reply:
Ah, yeah, it makes sense to just add a default in that case.